src/matrix-utils.ts

```diff
@@ -343,13 +343,18 @@ export async function createRoom(
 }
 
 // Returns a URL to that will load Element Call with the given room
-export function getRoomUrl(roomIdOrAlias: string): string {
+export function getRoomUrl(
+  roomIdOrAlias: string,
+  password: string = ""
+): string {
+  password = password === "" ? "" : "#?" + PASSWORD_STRING + password;
```

Comment:
Seems like we've still ended up with a '?' in here. If that's intentional, okay, but my suggestion was just to have "#password=foo" and then just parsing the hash with `URLSearchParams ` rather than writing our own parsing.